system/src/Grav/Console/Gpm/InstallCommand.php

```diff
@@ -322,6 +345,42 @@ public function installDependencies($dependencies, $type, $message, $required =
         }
     }
 
+    /**
+     * Given a $package, Grav will enable the package.
+     *
+     * @param array  $package_slug   The package to enable.
+     *
+     * @return bool Success of enabling a package.
+     * 
+     * @throws \Exception
+     */
+    public static function enablePackage($package_slug)
+    {
+        $grav = new Grav;
+
+        $config_file_path = USER_DIR . "config" . DS . "plugins" . DS;
+        $config_filename = $package_slug . '.yaml';
+        $config_file = File::instance($config_file_path . $config_filename);
+        
+        if (!is_writable($config_file->filename())) {
+            $message = "Could not save config for " . $package_slug . ". Is the config folder/ file writable?";
+            $grav::instance()['log']->error($message);
+            return false;
+        }
+
+        if ($config_file->exists()) {
+            // Load existing config file.
+            $config_data = Yaml::parse($config_file->content());
+        }
+
+        // Set 'enable' plugin option.
+        $config_data['enabled'] = true;
```

Comment:
This could fail if `$config_file` doesn't exist?

Reply:
No, if I change L366 to:
`if (!$config_file->writable($config_file->filename())) {`

Reply:
To step through the steps just prior:
if the config file is present, then the `Yaml::parse` loads its contents into the $config_data array.
Whether or not the config file isn't present, then this L377 will just create a new array (or edit the existing array) at $config_data with the enable key setting it to `true`